### PR TITLE
Docs 7655 update managing schedules topic with mule 4/timezone hn

### PR DESCRIPTION
**Pull request by hannanelson**
PDF: 
[17-after-Manage Schedules _ MuleSoft Documentation.pdf](https://github.com/mulesoft/docs-runtime-manager/files/4834300/17-after-Manage.Schedules._.MuleSoft.Documentation.pdf)

### CODE REVIEW

File: modules/ROOT/pages/managing-schedules.adoc [hunk 1]
```diff
@@ -1,130 +1,109 @@
...
+After deploying an app to CloudHub, view and manage the Scheduler elements in Runtime Manager:
 
 . Click *Applications* in the left pane, click the app, and then click *Schedules*:
 +
-image::ch-schedule-change.png[View Schedules]
+.The screenshot shows (*1*) the *Logs* option, (*2*) the *Last Run* column, and (*3*) the frequency
+link on the *Schedules* page.
+image::cloudhub-scheduler-rtm.png[Logs option and the Last Run column on the Schedules page]
++
+The *Last Run* column shows the date and time each scheduler element last ran.
```
Comment (MichelleZMY) [inline modules/ROOT/pages/managing-schedules.adoc]: keep "Scheduler" element capitalized?

File: modules/ROOT/pages/managing-schedules.adoc [hunk 2]
```diff
@@ -1,130 +1,109 @@
...
+You can click *Logs* in the left navigation menu to see more information about when the jobs started and ended scheduled runs.
 +
-. Select the elements to manage, then:
+Runtime Manager lists the Scheduler elements in the order that they appear in your application.
+. Select the elements to manage and then:
 
 ** Click *Run now* to override the schedule and run the job immediately.
 +
-** Click *Disable* to suspend a polling element.
+** Click *Disable* to suspend a scheduler element.
```
Comment (MichelleZMY) [inline modules/ROOT/pages/managing-schedules.adoc]: keep "Scheduler" element capitalized?

**Review by MichelleZMY — DISMISSED** [review]

File: modules/ROOT/pages/managing-schedules.adoc [hunk 3]
```diff
@@ -1,130 +1,109 @@
...
-== Considerations
+== Considerations and Limitations
 
 When using the *Schedules* tab, consider the following:
 
-* Schedulers are managed by CloudHub and are triggered in only one worker even if more are available.
-* Setting a polling schedule for less than a 10-second frequency between calls is not recommended.
-* If a scheduled job is not triggered one or more times because the application is down, CloudHub triggers the job as soon as the application restarts.
-* CloudHub scheduler reads the job configuration every time it runs.
+* CloudHub manages the Schedulers and triggers them in only one worker even if more are available.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-schedules.adoc]:
```suggestion
* CloudHub manages the Schedulers and triggers them in only one worker, even if more are available.
```

File: modules/ROOT/pages/managing-schedules.adoc [hunk 4]
```diff
@@ -1,130 +1,109 @@
...
+* All schedules are based in the UTC timezone, regardless of the geographic region where your 
+application is deployed.
+* The minimum recommended frequency is 10 seconds between calls.
+* If a scheduled job is not triggered because the application is down,
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-schedules.adoc]: I know that "down" is common; just being anal and wondering if we can use something without different meanings....

File: modules/ROOT/pages/managing-schedules.adoc [hunk 5]
```diff
@@ -1,130 +1,109 @@
...
-== Manage Schedules at Runtime
-
-To manage schedules in Runtime Manager:
+After deploying an app to CloudHub, view and manage the Scheduler elements in Runtime Manager:
 
 . Click *Applications* in the left pane, click the app, and then click *Schedules*:
 +
-image::ch-schedule-change.png[View Schedules]
+.The screenshot shows (*1*) the *Logs* option, (*2*) the *Last Run* column, and (*3*) the frequency
+link on the *Schedules* page.
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-schedules.adoc]: Is it okay that this is not blue?

File: modules/ROOT/pages/managing-schedules.adoc [hunk 6]
```diff
@@ -1,130 +1,109 @@
...
-image::ch-set-poll-freq.png[Set Polling Frequency]
+.The arrow shows the *Switch to advanced mode* link in the schedule editor.
+image::cloudhub-scheduler-frequency.png[Switch to advanced mode link in the schedule editor]
 +
-In the schedule editor, change the polling frequency of the element or deselect *Enabled* to disable the polling schedule.
+In the schedule editor, change the frequency of the schedule or deselect *Enabled* to disable the schedule.
 + 
-You can also <<Use Cron Expressions,use cron expressions>> to create more specific schedules, such as "every Thursday at 3:00 pm". 
+You can also switch to advanced mode to use cron expressions to create more specific schedules,
+such as "every Thursday at 3:00 pm".
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-schedules.adoc]:
```suggestion
such as `every Thursday at 3:00 pm`.
```

File: modules/ROOT/pages/managing-schedules.adoc [hunk 7]
```diff
@@ -1,130 +1,109 @@
...
 . Click *Update* to save changed settings or click *Cancel* to exit without saving.
-. Click *Logs* in the left navigation menu to verify that the job you selected ran as expected.
 
-=== Use Cron Expressions
+== Use Cron Expressions
 
-With cron expressions, you can create more complex polling frequencies, such as "every 5 minutes between 9:00 am and 10:00 on Fridays".
+With cron expressions, you can create more complex schedule frequencies,
+such as "every 5 minutes between 9:00 am and 10:00 on Fridays".
```
Comment (kbostrom) [inline modules/ROOT/pages/managing-schedules.adoc]:
```suggestion
such as `every 5 minutes between 9:00 am and 10:00 on Fridays`.
```

**Review by kbostrom — DISMISSED** [review]
Edited

File: modules/ROOT/pages/managing-schedules.adoc (hunk 5 above)
Comment (hannanelson) [reply modules/ROOT/pages/managing-schedules.adoc]: Good catch! The image wasn't showing, too.

File: modules/ROOT/pages/managing-schedules.adoc (hunk 6 above)
Comment (hannanelson) [reply modules/ROOT/pages/managing-schedules.adoc]: Not literal.

**Review by kbostrom — APPROVED** [review]
Reapproval

**Pull request merged**